### PR TITLE
Added release notes

### PR DESCRIPTION
**Pull request by tmcconechy**
Added release notes for 4.13.0

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,68 @@
...
+- `[Datagrid]` Fixed an error when dynamically adding context menus. ([#1216](https://github.com/infor-design/enterprise/issues/1216))
+- `[Datagrid]` Added an example of dynamic intermediate paging and filtering. ([#396](https://github.com/infor-design/enterprise/issues/396))
+- `[Dropdown]` Fixed alignment issues on mobile devices. ([#1069](https://github.com/infor-design/enterprise/issues/1069))
+- `[Datepicker]` Fixed incorrect assumptions, causing incorrect umalqura calendar calculations. ([#1189](https://github.com/infor-design/enterprise/issues/1189))
+- `[General]` Added the ability to stop renderLoop. ([#214](https://github.com/infor-design/enterprise/issues/214))
+- `[Datepicker]` Fixed an issue reselecting ranges with the date picker range option. ([#1197](https://github.com/infor-design/enterprise/issues/1197))
+- `[Editor]` Fixed bugs on IE with background color option ([#392](https://github.com/infor-design/enterprise/issues/392))
+- `[Colorpicker]` Fixed issue where the palette is no closed on enter key / click. ([#1050](https://github.com/infor-design/enterprise/issues/1050))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]: 'not closed'

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -5,11 +5,68 @@
...
+- `[Accordion]` Fixed issues with context menus on the accordion. ([#639](https://github.com/infor-design/enterprise/issues/639))
+- `[Searchfield]` Made no results appear unclickable ([#329](https://github.com/infor-design/enterprise/issues/329))
+- `[Datagrid]` Added an exaple of groups and paging. ([#435](https://github.com/infor-design/enterprise/issues/435))
```
Comment (davidcarlsonberg) [inline docs/CHANGELOG.md]: 'example'

File: docs/CHANGELOG.md (hunk 1 above)
Comment (clepore) [inline docs/CHANGELOG.md]:
```suggestion
- `[Colorpicker]` Fixed issue where the palette is not closed on enter key / click. ([#1050](https://github.com/infor-design/enterprise/issues/1050))
```

File: docs/CHANGELOG.md (hunk 2 above)
Comment (clepore) [inline docs/CHANGELOG.md]:
```suggestion
- `[Datagrid]` Added an example of groups and paging. ([#435](https://github.com/infor-design/enterprise/issues/435))
```

**Pull request merged**